portfolio/src/main/webapp/index.html

```diff
@@ -58,7 +58,22 @@ <h1 id="project">Servlet Tests</h1>
         <br>
         <input type="submit"/>
       </form>
+      
+      <p>Max Comments Shown:</p>
+      <select id="numComments">
+        <option name="1"> 1 </option>
+        <option name="2"> 2 </option>
+        <option name="3"> 3 </option>
+        <option name="4"> 4 </option>
+      </select>
+      <button onClick="fetchComments()">Load</button>
+
 
+      <form action="\delete-data" method="POST">
```

Comment:
`/delete-data` is more common. (A bit surprised `\delete-data` works - you learn something everyday :) )

Reply:
Also... would prefer `deleteComments` or `delete_comments`